### ISSUE TITLE
CLI cannot synthesize from a file with one line

### ISSUE THREAD

**Issue by jordanxlau** [EveryVoiceTTS/EveryVoice#690]
### Bug description

The EveryVoice CLI cannot synthesize from text files with just one line. No relevant error messages display, but no output files are generated. This is the case when synthesizing to any file type (wav, spec, etc.).

### How to reproduce the bug

```shell
# Where text.txt contains just one line
everyvoice synthesize from-text ... -f text.txt ...
```

### Error messages and logs

```
# None relevant
```

### Environment

<details>
<summary>Current environment</summary>
<pre>
EveryVoice Diagnostic information
EveryVoice version: 0.3.0
Python version: 3.12.10 (main, Apr  9 2025, 04:03:51) [Clang 20.1.0 ]
System: Linux 5.15.0-136-generic #147-Ubuntu SMP Sat Mar 15 15:53:30 UTC 2025 x86_64

*torch* modules installed using pip:
pytorch-lightning==2.5.1.post0
torch==2.3.1+cu118
torchaudio==2.3.1+cu118
torchinfo==1.8.0
torchmetrics==1.7.2

Other modules installed using pip:
absl-py==2.3.0
aiofiles==24.1.0
aiohappyeyeballs==2.6.1
aiohttp==3.12.4
aiosignal==1.3.2
annotated-types==0.7.0
anyio==4.9.0
anytree==2.13.0
arrow==1.3.0
attrs==25.3.0
audioread==3.0.1
black==24.10.0
certifi==2025.4.26
cffi==1.17.1
cfgv==3.4.0
chardet==5.2.0
charset-normalizer==3.4.2
chevron==0.14.0
click==8.1.8
clipdetect==0.1.4
coloredlogs==15.0.1
contourpy==1.3.2
coverage==7.8.2
cycler==0.12.1
decorator==5.2.1
deepdiff==8.5.0
diff-cover==9.3.1
distlib==0.3.9
dnspython==2.7.0
docopt==0.6.2
editdistance==0.8.1
einops==0.5.0
email-validator==2.2.0
et-xmlfile==2.0.0
-e file:///gpfs/fs5/nrc/nrc-fs1/ict/others/u/jol004/sandbox/EveryVoice
fastapi==0.115.12
ffmpy==0.5.0
filelock==3.18.0
flake8==7.2.0
fonttools==4.58.1
frozenlist==1.6.0
fsspec==2025.5.1
g2p==2.2.1
gitlint-core==0.19.1
gradio==5.31.0
gradio-client==1.10.1
grapheme==0.6.0
greenlet==3.2.2
groovy==0.1.2
grpcio==1.71.0
h11==0.16.0
hf-xet==1.1.2
httpcore==1.0.9
httpx==0.28.1
huggingface-hub==0.32.2
humanfriendly==10.0
identify==2.6.12
idna==3.10
iniconfig==2.1.0
ipatok==0.4.2
isort==6.0.1
jinja2==3.1.6
joblib==1.5.1
jsonschema==4.24.0
jsonschema-specifications==2025.4.1
kiwisolver==1.4.8
librosa==0.9.2
lightning==2.5.1.post0
lightning-utilities==0.14.3
llvmlite==0.44.0
loguru==0.6.0
lxml==5.4.0
markdown==3.8
markdown-it-py==3.0.0
markupsafe==3.0.2
matplotlib==3.9.4
mccabe==0.7.0
mdurl==0.1.2
merge-args==0.1.5
mpmath==1.3.0
multidict==6.4.4
munkres==1.1.4
mypy==1.16.0
mypy-extensions==1.1.0
networkx==3.5
nltk==3.9.1
nodeenv==1.9.1
numba==0.61.2
numpy==1.26.4
nvidia-cublas-cu11==11.11.3.6
nvidia-cuda-cupti-cu11==11.8.87
nvidia-cuda-nvrtc-cu11==11.8.89
nvidia-cuda-runtime-cu11==11.8.89
nvidia-cudnn-cu11==8.7.0.84
nvidia-cufft-cu11==10.9.0.58
nvidia-curand-cu11==10.3.0.86
nvidia-cusolver-cu11==11.4.1.48
nvidia-cusparse-cu11==11.7.5.86
nvidia-nccl-cu11==2.20.5
nvidia-nvtx-cu11==11.8.86
openpyxl==3.1.5
orderly-set==5.4.1
orjson==3.10.18
packaging==24.2
pandas==2.2.3
panphon==0.20.0
pathspec==0.12.1
pep440==0.1.2
pillow==11.2.1
platformdirs==4.3.8
playwright==1.52.0
pluggy==1.6.0
pooch==1.8.2
pre-commit==4.2.0
prompt-toolkit==3.0.51
propcache==0.3.1
protobuf==4.25.8
pycodestyle==2.13.0
pycountry==22.3.5
pycparser==2.22
pydantic==2.7.4
pydantic-core==2.18.4
pydub==0.25.1
pyee==13.0.0
pyflakes==3.3.2
pygments==2.19.1
pympi-ling==1.70.2
pyparsing==3.2.3
pysdtw==0.0.5
pytest==7.4.4
python-dateutil==2.9.0.post0
python-multipart==0.0.20
python-slugify==5.0.0
pytz==2025.2
pyworld-prebuilt==0.3.4.4
pyyaml==6.0.2
questionary==1.10.0
readalongs==1.2.1
referencing==0.36.2
regex==2024.11.6
requests==2.32.3
resampy==0.4.3
rich==14.0.0
rpds-py==0.25.1
ruff==0.11.12
safehttpx==0.1.6
scikit-learn==1.6.1
scipy==1.15.3
semantic-version==2.10.0
setuptools==80.9.0
sh==2.2.2
shellingham==1.5.4
simple-term-menu==1.5.2
six==1.17.0
sniffio==1.3.1
soundfile==0.13.1
soundswallower==0.6.5
starlette==0.46.2
sympy==1.14.0
tabulate==0.9.0
tensorboard==2.19.0
tensorboard-data-server==0.7.2
termcolor==2.3.0
text-unidecode==1.3
threadpoolctl==3.6.0
tomlkit==0.13.2
tqdm==4.67.1
typer==0.16.0
types-python-dateutil==2.9.0.20250516
types-pyyaml==6.0.12.20250516
types-requests==2.32.0.20250515
types-setuptools==80.9.0.20250529
types-tabulate==0.9.0.0
types-tqdm==4.67.0.20250516
typing-extensions==4.13.2
tzdata==2025.2
unicodecsv==0.14.1
urllib3==2.4.0
uvicorn==0.34.2
virtualenv==20.31.2
wcwidth==0.2.13
websockets==15.0.1
webvtt-py==0.4.2
werkzeug==3.1.3
yarl==1.20.0
yaspin==3.1.0
</pre>
</details>
